pocket_coffea/executors/executors_casa.py

```diff
@@ -0,0 +1,42 @@
+from pocket_coffea.executors.executors_base import ExecutorFactoryABC
+from pocket_coffea.executors.executors_base import IterativeExecutorFactory, FuturesExecutorFactory
+from coffea import processor as coffea_processor
+    
+
+class DaskExecutorFactory(ExecutorFactoryABC):
+
+    def __init__(self, run_options, outputdir, **kwargs):
+        self.outputdir = outputdir
+        super().__init__(run_options, **kwargs)
+        
+    def setup(self):
+        ''' Start the DASK cluster here'''
+        # At coffea-casa we have preconfigured Dask HTCondor cluster for you, please just use it available at tls://localhost:8786
+        from distributed import Client
+        self.dask_client = Client("tls://localhost:8786")
```

Comment:
If this port always the same? Is it needed to put it in the default parameters?

Reply:
It is always the same for each user